src/otp-input/otp-input.tsx

```diff
@@ -226,6 +232,11 @@ export const OtpInput = ({
                 role="group"
                 aria-label={`${numOfInput}-digit OTP input field`}
             >
+                {prefix && (
+                    <Typography.HeadingSM style={{ textWrap: "nowrap" }}>
```

Comment:
1. SM is 22px which seems kind of huge, default font size is 18px for most themes, checking if this is part of the design?
2. instead of inlining the style, declare a styled component
3. set `as` prop so that this doesn't render as a `h5` element

Reply:
PA mock up uses SM. When looking on storybook with LSG theme, the size when SM looks to correspond with the OTP input box.

noted on points 2 and 3